modules/ee/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/EE",
+	"url": "https://www.dndbeyond.com/sources/dnd/ee",
```

Comment:
This URL 404's, and I can't find anything that seems to correspond to this on the DDB site...